server/resources/js/app/device_metric_graphs.js

```diff
@@ -113,10 +113,10 @@ class MetricGrapher {
     this.memoryUsageHistory_.push(this.metrics.memUsage / 1024.0);
     this.memoryUsageHistory_.shift();
 
-    this.networkSentHistory_.push(this.metrics.netSentPerSec);
+    this.networkSentHistory_.push(this.metrics.netSentPerSec / 1024.0);
```

Comment:
why 1024? May be good to have this as a constant and comment about why it's 1024